andela/team-odd-bn-frontend
# #170574394 user should be able to verify email

#### What does this PR do?
This PR adds email verification page after user signs up
#### Description of Task to be completed?
- A user is able to resend a verification email after signup
- A user is able to navigate directly to the app after email verification
#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-frontend`
- `git checkout ft-email-verification-page-170574394`
- `npm install` packages
- `npm run dev` to start the development server
- It should open your default browser automatically
- Navigate to `localhost:4000/resend-email` to access the page directly
#### Any background context you want to provide?
Previously, the users used to validate their email using the backend part
#### What are the relevant pivotal tracker stories?
[#170574394](https://www.pivotaltracker.com/story/show/170574394)
#### Screenshots (if appropriate)
<img width="1440" alt="Screen Shot 2020-01-09 at 12 53 25" src="https://user-images.githubusercontent.com/35301129/72061696-133bbf80-32df-11ea-9fd5-a0fddbb5864c.png">

#### Questions:
N/A
